CAIRN Info.js

```diff
@@ -116,6 +135,143 @@ var testCases = [
+				"creators": [
+					{
+						"firstName": "Xavier",
+						"lastName": "Landes",
+						"creatorType": "author"
+					}
+				],
+				"date": "2014-05-13",
+				"ISSN": "0048-8046",
+				"issue": "1",
+				"language": "fr",
+				"libraryCatalog": "Cairn.info",
+				"pages": "65-92",
+				"publicationTitle": "Revue européenne des sciences sociales",
+				"url": "http://www.cairn.info/resume.php?ID_ARTICLE=RESS_521_0065",
+				"volume": "52",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					},
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					"bénéfices sociaux",
+					"libre accès",
+					"publications académiques",
+					"recherche",
+					"État"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.cairn.info/resume.php?ID_ARTICLE=RHIS_121_0049",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Le mouvement pétitionnaire pour la restauration d'Henri V (automne 1873-hiver 1874). Tactique politique et expression d'un légitimisme populaire",
+				"creators": [
+					{
+						"firstName": "Éric",
+						"lastName": "Derennes",
+						"creatorType": "author"
+					}
+				],
+				"date": "2012-04-17",
+				"ISSN": "0035-3264",
+				"issue": "661",
+				"language": "fr",
+				"libraryCatalog": "Cairn.info",
+				"pages": "49-99",
+				"publicationTitle": "Revue historique",
+				"url": "http://www.cairn.info/resume.php?ID_ARTICLE=RHIS_121_0049",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					},
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [
+					"assemblée nationale",
+					"député",
+					"légitimisme",
+					"pétition",
+					"restauration",
+					"royaliste"
+				],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
+	},
+	{
+		"type": "web",
+		"url": "http://www.cairn.info/resume.php?ID_ARTICLE=RFS_523_0537",
+		"items": [
+			{
+				"itemType": "journalArticle",
+				"title": "Transformation de l'État ou changement de régime ? De quelques confusions en théorie et sociologie de l'État",
+				"creators": [
+					{
+						"firstName": "Paul Du",
+						"lastName": "Gay",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "Alan",
+						"lastName": "Scott",
+						"creatorType": "author"
+					}
+				],
+				"date": "2011-10-01",
+				"ISSN": "0035-2969",
+				"issue": "3",
+				"language": "fr",
+				"libraryCatalog": "Cairn.info",
+				"pages": "537-557",
+				"publicationTitle": "Revue française de sociologie",
+				"shortTitle": "Transformation de l'État ou changement de régime ?",
+				"url": "http://www.cairn.info/resume.php?ID_ARTICLE=RFS_523_0537",
+				"volume": "52",
+				"attachments": [
+					{
+						"title": "Full Text PDF",
+						"mimeType": "application/pdf"
+					},
+					{
+						"title": "Snapshot"
+					}
+				],
+				"tags": [],
+				"notes": [],
+				"seeAlso": []
+			}
+		]
```

Comment:
So all of these new test cases test something different? Could you comment (here on github) on what they test?